Octokit.Tests/Clients/MetaClientTests.cs

```diff
@@ -23,7 +23,8 @@ public async Task RequestsTheMetadataEndpoint()
                      new[] { "1.1.6.1/24", "1.1.6.2/24" },
                      new[] { "1.1.7.1", "1.1.7.2" },
                      new[] { "1.1.8.1/24", "1.1.8.2/24" },
-                     new[] { "1.1.9.1", "1.1.9.2" }
+                     new[] { "1.1.9.1", "1.1.9.2" },
+                     null
```

Comment:
Do you mind explaining the addition of the `null` here? I'm not sure I follow.

Reply:
That was just to satisfy the required constructor arguments. I left it as `null` here because `installed_version` isn't present on calls to github.com. However, I'm happy to change it to `3.7.0` if you'd prefer to have all the values filled in. I could also make that constructor parameter optional if that's preferable.

Reply:
Hmm...I would probably prefer an explicit argument there, but I'm unable to articulate a good case for why. Matching other behavior is probably our best bet.

Reply:
Makes sense! I've updated it to an actual value